src/tron/tronTypes.ts

```diff
@@ -73,6 +73,34 @@ export interface TronTransferParams {
   note?: string
 }
 
+const asResource = asValue('BANDWIDTH', 'ENERGY')
+type Resource = ReturnType<typeof asResource>
+
+export interface TronFreezeAction {
+  name: 'add'
```

Comment:
Please call this `type` instead of `name`.

Reply:
Ok will update